### PR TITLE
fix #75405 news.abplive.com

### LINKED ISSUE #75405: news.abplive.com
### Issue URL (Ads)
 [https://news.abplive.com/lifestyle/watch-viral-video-of-flying-dosa-mumbai-man-s-technique-of-serving-food-leaves-netizens-jaw-dropped-1444958/amp](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fnews.abplive.com%2Flifestyle%2Fwatch-viral-video-of-flying-dosa-mumbai-man-s-technique-of-serving-food-leaves-netizens-jaw-dropped-1444958%2Famp)
### Comment
Username: `@J-khatri`
> 
> --
> The information below is automatically generated. Do not delete it.
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/disable-amp/1.0/disable-amp.meta.js
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.meta.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Android 10
AdGuard product: | AdGuard for Android v3.6.1 (11)
Browser: | Chrome
AdGuard mode: | VPN
Filtering quality: | High-quality
HTTPS filtering: | enabled
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/> Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>EasyList, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy<br/><br/><b>Annoyances:</b><br/> I don't care about cookies, <br/>Adblock Warning Removal List, <br/>AdGuard Annoyances, <br/>Fanboy's Annoyances, <br/>Web Annoyances Ultralist, <br/>EasyList Cookie List<br/><br/><b>Security:</b><br/> NoCoin Filter List, <br/>Online Malicious URL Blocklist, <br/>Spam404<br/><br/><b>Other:</b><br/> AdGuard DNS<br/><br/><b>Language-specific:</b><br/> AlleBlock, <br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by stanislav-atr**
#75405

### CODE REVIEW

**Comment by Sergey-Lyapin** [comment]
share buttons are missed
Screenshot 2021-03-01 at 13 27 20

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -78,6 +78,7 @@ ansa.it##.bt-news-flip
 ansa.it##.social-menu li.button-wrap
 slovoidilo.ua##.wrap_addthis_inline_bottom
 bykvu.com##.article-share-box
+abplive.com##.mt-4 > a[href^="https://www.youtube.com/"]
```
Comment (Sergey-Lyapin) [inline SocialFilter/sections/specific.txt:81]: I think this rule is better:
`abplive.com##amp-img[src="https://cdn.abplive.com/images/uncut-band-amp-article.png"]`

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]
Some leftovers are missed and please check also non-amp version of this site
<details>
<summary>leftovers</summary>



</details>

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -290,6 +290,10 @@ uwatchfree.tech##a[href*="https://www.topcpmnetwork.com/"]
 ||hiphopza.com/wp-content/uploads/*/ad.gif
 ||wp.com/hiphopza.com/wp-content/uploads/*/ad.gif$domain=hiphopza.com
 ||cdjapan.co.jp/aff/tmpl/big_banner_728x90i.js
+abplive.com##a[href^="http://onelink.to"][rel="nofollow"]
```
[the remark this answers is not in the document]
Comment (Alex-302) [reply EnglishFilter/sections/specific.txt:293]: or `amp-img[alt="get app pic"]`

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply SocialFilter/sections/specific.txt:81]: or `a[href*="?sub_confirmation="]`

**Review by Alex-302 — APPROVED** [review]

**Comment by Sergey-Lyapin** [comment]
@stanislav-atr what about https://github.com/AdguardTeam/AdguardFilters/pull/75816#issuecomment-787838792 https://github.com/AdguardTeam/AdguardFilters/pull/75816#pullrequestreview-600596257?

**Pull request merged**